README.md

```diff
@@ -84,170 +98,172 @@ This code will generate the following terminal output on success:
+- The client will authenticate by providinf their client credentials through the `--client_id` and `--client_secret` program arguments. In this case **a token file must also be supplied even if it is a blank file**. Client will check file to see if it is a valid token, if it isn't it will refresh automatically the token and fill the file with a valid token. In this case, client_id and client_secret fields are also required.
 
-{
-  "access_token": "EXAMPLE_ACCESS_TOKEN",
-  "expiration_time": 1678453328
-}
 
-### Automatically Refresh Service Token
+#### Client id and secret
 
-This repository optionally implements an automatic token update. To use it, you must specify your credentials (find them in the Client Credentials section of the [user dashboard](https://dashboard.speechcenter.verbio.com)).
 
-You must also specify a token file, where the token will be stored and updated in case it is invalid or expired.
+```
+--client-id arg      Client id for token refresh (default: "")
+```    
 
-**Example**
-```shell
-python3 recognizer_stream.py --client-id="your-client-id" --client-secret="your-client-secret"
- --audio-file file.wav --topic GENERIC --language en-US --host us.speechcenter.verbio.com --token token.file --asr-version V1 --label project1
+``` 
+--client-secret arg  Client secret for token refresh (default: "")
+``` 
+
+`client-id` and `client-secret` fields are required for automatic token refreshal. The arguments need to be written inline with no quotes for each field. Please see the '**Automatically Refresh Service Token**' section above for specific details.
+
+#### Host
 
+
+```
+-H, --host arg
 ```
 
-**WARNING**
+URL of the host where the request will be sent. Main endpoints will be expanded as the product is deployed in different regions. Please use **us.speechcenter.verbio.com** as the host for CSR requests.
 
-Please note that due to a limitation of the command line argument parser, the `client_id` and `client_secret` arguments MUST be written in the following format since they might contain hyphens.
+#### Labels
 
-```text
---client-id="your-client-id"
-           ^
 ```
-> Note the usage of the `=` sign
+-L, --label arg
+```
 
-Not defining the arguments like this will yield an error.
+This option allows for a one word argument to be sent so that the speech transcription request is billed as part of a particular project for the customer. The argument will be a one word name that will classify the request under that project.
 
-### Client flags
+- **Project name must only consist of one word.**
+- **Argument must be the same each time for the same project. If there is a typo another project will be created.**
+- **There is no limit on the amount of projects that can be created.**
 
-#### Synthesizer (GRPC)
 
-##### Audio file
+## Speech synthesizer (TTS)
```

Comment:
TTS means Text-to-Speech Synthesis, so I would expand the acronym completely.

Reply:
Definitely, I will fix it and commit a new change.